### PR TITLE
Sipm response 

### PR DESCRIPTION
**Pull request by carmenromo**
This PR adds several functions to include a double exponential distribution in the sipm response in order to take into account the electronics effect.

### CODE REVIEW

**Comment by carmenromo** [comment]
tof_electronics_dist_example

The double exponential distribution is showed in the plot, in comparison to the normal tof case.

File: antea/elec/tof_functions_test.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+from hypothesis  import given
+
+import tof_functions as tf
+from antea.io.mc_io import load_mcTOFsns_response
+
+
+a = st.floats(min_value=1, max_value=100000)
+b = st.floats(min_value=2, max_value=100000)
+l = st.lists(st.integers(min_value=1, max_value=10000), min_size=2, max_size=1000)
+
```
Comment (paolafer) [inline antea/elec/tof_functions_test.py]: Could you add a brief description of what each test does?

Comment (carmenromo) [reply antea/elec/tof_functions_test.py]: Sure!

**Comment by viherbos** [comment]
Code related to signal convolution checked.
Everything looks consistent with the original "toy" code.
Has energy resolution been tested after normalization of the spe response?

**Comment by carmenromo** [comment]
Not yet, I will test it

**Comment by paolafer** [comment]
I think that it would be good to run our usual analysis using this function and compare the results.

**Comment by carmenromo** [comment]
Yes, I agree

**Comment by paolafer** [comment]
Also, I would like to open a repository for demonstration notebooks and the usage of this function could be a nice first example! A notebook that apply the function and produces the plots you show above, for instance.

**Comment by carmenromo** [comment]
Perfect, I have it in my own repository in fact, so I will make a new repository and move it there.

**Comment by paolafer** [comment]
I'll ask José María to add a new repository to nextic for ANTEA notebooks and you can push it there.

**Pull request merged**